### PR TITLE
url_path() update

### LINKED ISSUE #26: Fix url_path method for new data location
url_path is returning the url path as:
`https://docker-dev03.pcic.uvic.ca/twitcher/ows/proxy/thredds/{identifier}/datasets/storage/data/{sub_filepath}`
url_path should return the url as:
`https://docker-dev03.pcic.uvic.ca/twitcher/ows/proxy/thredds/{identifier}/datasets/storage/data/projects/comp_support/daccs/test-data/{sub_filepath}` according to the new location

### PR DESCRIPTION
**Pull request by cairosanders**
Fixes Urls in `url_path` in `testing.py` and related tests.

Resolves #26

### CODE REVIEW

**Review by nikola-rados — APPROVED** [review]
LGTM!

File: tests/test_testing.py [hunk 1]
```diff
@@ -19,16 +19,23 @@ def test_local_path(nc_file):
...
-    ("nc_file", "url_type"), [(nc_file, "opendap"), (nc_file, "http"),]
+    ("nc_file", "url_type", "sub_dir"),
+    [
+        (nc_file, "opendap", "daccs"),
+        (nc_file, "http", "daccs"),
+        (nc_file, "opendap", "climate_explorer_data_prep"),
+    ],
 )
-def test_opendap_path(nc_file, url_type):
+def test_opendap_path(nc_file, url_type, sub_dir):
```
Comment (sum1lim) [inline tests/test_testing.py]: I think I forgot to change the name of the test to `test_url_path`. Could you change it in this PR?

**Review by sum1lim — APPROVED** [review]
LGTM

File: tests/test_testing.py (hunk 1 above)
Comment (cairosanders) [reply tests/test_testing.py]: Oh ya! I noticed that and forgot to change it too.

**Pull request merged**